sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -474,6 +474,43 @@ public async Task StartRecognizeCustomFormsWithLabelsCanParseMultipageForms(bool
             }
         }
 
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task StartRecognizeCustomFormsWithLabelsCanParseDifferentTypeOfForm(bool useStream)
+        {
+            var client = CreateInstrumentedFormRecognizerClient();
+            RecognizeCustomFormsOperation operation;
+
+            // Use Form_<id>.<ext> files for training with labels.
+
+            await using var trainedModel = await CreateDisposableTrainedModelAsync(useTrainingLabels: true);
+
+            // Attempt to recognize a different type of form: Invoice_1.pdf. This form does not contain all the labels
+            // the newly trained model expects.
+
+            if (useStream)
```

Comment:
I don't think for the purpose of this test the form source matters, as we care about the deserialize section and both methods hit it. Consider using local file only

Reply:
I thought about using only the local file, but wasn't sure whether we could safely assume that it would be enough or not. I like your way of thinking, though. If we only care about the response, no need to test the URI as well, since it's expected to change only the request we're sending.